cms/admin/placeholderadmin.py

```diff
@@ -283,7 +283,6 @@ def add_plugin(self, request):
         plugin = CMSPlugin(language=language, plugin_type=plugin_type, position=position, placeholder=placeholder)
 
         if parent:
-            plugin.position = CMSPlugin.objects.filter(parent=parent).count()
```

Comment:
As it's written there it's just plain wrong, and yes, it relies on position being calculated on save